Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="denys.kotovskiy@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
```

Comment:
The `COPY requirements.txt requirements.txt` command should be changed to `COPY requirements.txt /app/` to ensure consistency and clarity in the file structure.